ApollosProject/apollos-apps
# test/no-video-routing

## ✏️ Description




## 🔬 To Test
1. Go to a content series that has no videos in it (Item 5 in apollos_demo is a good one)
2. Press the hardware back button
3. It should send you back the the previous screen instead of exiting out of the app

## 📸 Screenshots


## ⚠️ To-do Before Merge



## 📝 Checklist
